index.js

```diff
@@ -20,8 +21,29 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
-  })
+    Recipe.create({title:'Cold Chocolate Drink', level:'Easy Peasy', ingredients: ['oat milk','raw cacao'], cuisine:'Raw Foods', dishType: 'drink', duration: 2, creator: 'LWB'})
+      .then(recipe => console.log(`The recipe was saved and its title is: ${recipe.title}`))
+      .catch(error => console.log(`An error happened: ${error}`));
+    
+    Recipe.insertMany(data)
+      .then (recipes => recipes.forEach(recipe => console.log(`Title: ${recipe.title}`)))
+      .catch (error => console.log(`Error occurred adding recipes to database:${error}`));
+      
+    Recipe.findOneAndUpdate({title:'Rigatoni alla Genovese'},{$set:{duration:100}}, {new:true})
+      .then (success => console.log(`Recipe successfully updated`))
+      .catch( error => console.log(`Error occurred updating recipe: ${error}`));
+    
+    Recipe.deleteOne({title:'Carrot Cake'})
+      .then (deleted => console.log(`Recipe successfully deleted`))
+      .catch(error => console.log(`Error occurred deleting recipe: ${error}`));
+    })
+
+    /*.then(() => {
```

Comment:
In here, you're treating this section as a chained promise, and since you're not returning anything your .then() is rendered useless in this case; it would work if either you chain your .then()s/make promise.all() and start returning your methods inside your .then()s, or if you insert the command to close the connection in the final CRUD method, OR if you just call it as another block of code after your CRUD methods. All of these options can be seen in the solutions posted in the class sheet. If you need any extra explanation, let me know!